Sources/Components.swift

```diff
@@ -0,0 +1,119 @@
+                                                     forKey: .parameters) ?? [:]
+        self.securitySchemes = try values.decodeIfPresent([String: Reference<SecuritySchemaBuilder>].self,
+                                                          forKey: .securitySchemes) ?? [:]
+        self.examples = try values.decodeIfPresent([String: Reference<ExampleBuilder>].self,
+                                                   forKey: .examples) ?? [:]
+        self.requestBodies = try values.decodeIfPresent([String: Reference<RequestBodyBuilder>].self,
+                                                        forKey: .requestBodies) ?? [:]
+        self.headers = try values.decodeIfPresent([String: Reference<HeaderBuilder>].self,
+                                                  forKey: .headers) ?? [:]
+        self.links = try values.decodeIfPresent([String: Reference<LinkBuilder>].self,
+                                                forKey: .links) ?? [:]
+        self.callbacks = try values.decodeIfPresent([String: Reference<CallbackBuilder>].self,
+                                                    forKey: .callbacks) ?? [:]
+    }
+}
+
+extension ComponentsBuilder: Builder {
+    typealias Building = Components
+    
+    func build(_ swagger: SwaggerBuilder) throws -> Components {
+        let schemas = try self.schemas.mapValues {
+            try SchemaBuilder.resolve(swagger, reference: $0)
+        }
+        let responses = try self.responses.mapValues {
+            try ResponseBuilder.resolve(swagger, reference: $0)
+        }
+        let parameters = try self.parameters.mapValues {
+            try ParameterBuilder.resolve(swagger, reference: $0)
+        }
+        let securitySchemes = try self.securitySchemes.mapValues {
+            try SecuritySchemaBuilder.resolve(swagger, reference: $0)
+        }
+        let examples = try self.examples.mapValues {
+            try ExampleBuilder.resolve(swagger, reference: $0)
+        }
+        let requestBodies = try self.requestBodies.mapValues {
+            try RequestBodyBuilder.resolve(swagger, reference: $0)
+        }
+        let headers = try self.headers.mapValues {
+            try HeaderBuilder.resolve(swagger, reference: $0)
+        }
+        let links = try self.links.mapValues {
+            try LinkBuilder.resolve(swagger, reference: $0)
+        }
+        let callbacks = try self.callbacks.mapValues {
+            try CallbackBuilder.resolve(swagger, reference: $0)
+        }
+        return Components(schemas: schemas,
+                          responses: responses,
+                          parameters: parameters,
+                          examples: examples,
+                          requestBodies: requestBodies,
+                          headers: headers,
+                          securitySchemes: securitySchemes,
+                          links: links,
+                          callbacks: callbacks)
+    }
+}
```

Comment:
### Reusable [Components Object](https://swagger.io/specification/#componentsObject)
Used in **Swagger** field 'components'